### PR TITLE
rto_node: add stop service

### PR DESCRIPTION
**Pull request by TarikViehmann**
This service may be used to stop any cmd velocity commands sent to the omni drive.

This is useful in scenarios where a sudden stop, e.g., emergency stop, needs to be realized.

### CODE REVIEW

File: rto_node/src/OmniDriveROS.cpp [hunk 1]
```diff
@@ -70,4 +77,14 @@ void OmniDriveROS::setMaxMin(double max_linear_vel, double min_linear_vel, doubl
...
 	max_angular_vel_ = max_angular_vel;
 	min_angular_vel_ = min_angular_vel;
-}
\ No newline at end of file
+}
+
+bool OmniDriveROS::handleSetOmniDriveEnabled(const std::shared_ptr<rto_msgs::srv::SetOmniDriveEnabled::Request> request,
+                                    std::shared_ptr<rto_msgs::srv::SetOmniDriveEnabled::Response> response)
+{
+    enabled_ = request->enabled;
```
Comment (borsesaurabh2022) [inline rto_node/src/OmniDriveROS.cpp]: enabled_ = request->enable, change the request variable name.

**Review by borsesaurabh2022 — DISMISSED** [review]
looks good to me, make this change and then we can merge.

File: rto_node/src/OmniDriveROS.cpp (hunk 1 above)
Comment (TarikViehmann) [reply rto_node/src/OmniDriveROS.cpp]: oh true, thank you!

**Review by borsesaurabh2022 — APPROVED** [review]
Looks good to me, great work !

**Pull request merged**